src/test/scala/mesosphere/marathon/api/v2/AppsResourceTest.scala

```diff
@@ -341,6 +342,89 @@ class AppsResourceTest extends MarathonSpec with MarathonActorSupport with Match
     JsonTestHelper.assertThatJsonString(response.getEntity.asInstanceOf[String]).correspondsToJsonOf(expected)
   }
 
+  /*
+  test("Create a new app with IP/CT with virtual network foo w/ Docker") {
```

Comment:
I started hacking on the group manager tests because this seemed futile. Was hoping that I was just missing something simple but from your explanation it seems I'm pushing the limits of what this suite of unit tests is aimed at.

There's probably a case to make for tests that live somewhere in between simpler unit testing and full-fledged integration testing. For example I want to round-trip test from a JSON create-app request down through an actual group manager, without the hassle of setting up a mesos environment that supports overlay networking.

We can probably remove this commented test for now since I've tried to augment via group manager tests, even though it's less ideal IMHO.